docs/js-guide/static-files.md

```diff
@@ -52,53 +52,10 @@ things get REAL slow. Plus if you want to use any javascript compilers like
 
 Make sure your `localsettings.py` file has the following set:
 ```
-LESS_DEBUG = True
-LESS_WATCH = True  # if you want less.js to watch for changes and compile on the fly!
 COMPRESS_ENABLED = False
 COMPRESS_OFFLINE = False
 ```
 
-### 2. On-the-fly server-side compression, with the cache refreshed as changes are made.
```

Comment:
The header above now calls this section "Client side js-based `less` compilation." which isn't true. This header is the correct one, right? The pros and cons list above also refers to less.js and can be deleted.

Reply:
Oh thanks. This header is wrong too because False / False doesn't _compress_ anything, it just compiles less to (uncompressed) css. But you're right I'll change the previous header.